src/components/ToggleView/ToggleSelector.js

```diff
@@ -19,6 +19,7 @@ export const TOGGLE_VIEWS = {
 const mobileBreakpoint = '450px';
 
 const ToggleSelector = ({ className, showTooltip }) => {
+  showTooltip = true;
```

Comment:
should this be here? this means it pops up even if you're already on the alternate view

Reply:
nope 🤡